ipn/ipnlocal/serve.go

```diff
@@ -430,9 +348,7 @@ func (b *LocalBackend) maybeLogServeConnection(destPort uint16, srcAddr netip.Ad
 		}
 	}
 
-	for _, stream := range streamers {
-		stream(log)
-	}
+	b.send(ipn.Notify{RequestAccessLog: log})
```

Comment:
so this whole Line 335-351 is wasteful in the case that nobody is connected to the bus and wanting streaming logs.

that's why I was talking about keeping a refcount of log watchers. You can then skip these allocs & WhoIs lookup & b.send if nobody's streaming logs.

Reply:
Per our chat, we're punting on logs in this PR